src/app.ts

```diff
@@ -0,0 +1,67 @@
+interface UserAccount{
+    m_firstName:string,
+    m_lastName:string,
+    m_bankNumber:number
+}
+
+
+
+class BankAccount implements UserAccount{
+    
+    private _m_firstName:string;
```

Comment:
dont use SNAKE CASE in class properties 

read here

https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841